### ISSUE TITLE
.ruby-version conflict

### ISSUE THREAD

**Issue by neonmate** [makandra/katapult#15]
I saw following conflict when saying `katapult new $APPLICATION_NAME`:

```
> Generating katapult basics ...
       force  .gitignore
    conflict  .ruby-version
Overwrite /Users/neonmate/Projects/$APPLICATION_NAME/.ruby-version? (enter "h" for help) [Ynaqdh] 
```

Before: `.ruby-version` is empty
Afterwards (Saying Y): It is 2.3.0

I also would have expected this is  `2.4.1` now.

**Comment by neonmate** [comment]
I can't reproduce this issue with the new version anymore. But maybe this can be adjusted:

```
index 005119b..8e8299d 100644
--- a/.ruby-version
+++ b/.ruby-version
@@ -1 +1 @@
-2.4.1
+2.4.2
```

**Comment by codener** [comment]
Thanks for investigating! Meanwhile I have fixed the collision and upgraded to Ruby 1.5.0.
